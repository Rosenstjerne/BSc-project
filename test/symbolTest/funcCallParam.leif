var int a, b;

function int add(int x, int y) {
    return x + y;
}

a = 5;
b = 10;

print(add(a, b)); # Should print 15