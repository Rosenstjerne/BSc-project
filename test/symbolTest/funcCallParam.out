function null main() {
	var int a, b;
	
	function int add(int x, int y) {
		return (x + y);
		
	}
	
	a = 5;
	b = 10;
	print(add(a, b));
	
}
